ferrocene/tools/blanket/Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "blanket"
-edition = "2024"
+edition.workspace = true
```

Comment:
oh this is extremely cute, i love that we can do this now :) a future PR could extend this to the dependencies too